app/collector/druid_collector.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+from sqlalchemy.engine import create_engine
+import sqlalchemy as db
+
+from app.collector.sql_alchemy_collector import SqlAlchemyCollector
+
+class DruidCollector(SqlAlchemyCollector):
+# Class to implement methods to collect data in DRUID.
+
+    def get_databases(self):
+    # Return just the Default value.
+
+        database_list = ['default']
+        return database_list
+
+    def get_schemas(self, database_name: str):
```

Comment:
To discuss: do we need to return a schema even if the provider doesn't support it? The entity model supports tables without schema, directly associated to the database.